src/components/applicationComponent.tsx

```diff
@@ -327,19 +327,12 @@ class ManageApplicationContainer extends React.Component<
           isMulti={true}
           creatable={true}
           options={[
-            {
-              label: IEthnicity.AFRO_AMER,
-              value: IEthnicity.AFRO_AMER,
-            },
-            {
-              label: IEthnicity.ASIAN_PI,
-              value: IEthnicity.ASIAN_PI,
-            },
-            {
-              label: IEthnicity.EUROPEAN,
-              value: IEthnicity.EUROPEAN,
-            },
+            { label: IEthnicity.AFRO_AMER, value: IEthnicity.AFRO_AMER },
```

Comment:
Does this make the linter mad

Reply:
I ran the linter at the end and it didn't fix it so I guess not?